src/jit/importer.cpp

```diff
@@ -13905,12 +13905,13 @@ bool Compiler::impReturnInstruction(BasicBlock *block, int prefixFlags, OPCODE &
         impAppendTree(op2, (unsigned)CHECK_SPILL_NONE, impCurStmtOffs);
 
         // There are cases where the address of the implicit RetBuf should be returned explicitly (in RAX).  
-#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        // System V ABI requires to return the implicit return buffer explicitly (in RAX).
+#if defined(_TARGET_AMD64_)
+        // x64 (System V and Win64) calling convention requires to 
+        // return the implicit return buffer explicitly (in RAX).
         // Change the return type to be BYREF.  
         op1 = gtNewOperNode(GT_RETURN, TYP_BYREF, gtNewLclvNode(info.compRetBuffArg, TYP_BYREF));
-#else // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        // In case of Windows AMD64 the profiler hook requires to return the implicit RetBuf explicitly (in RAX).  
+#else // defined(_TARGET_AMD64_)
```

Comment:
(Minor) //!_TARGET_AMD64_

Similarly #endif associated with this #else should be changed to
//!_TARGET_AMD64_